### PR TITLE
Various python cleanups

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

**Comment by sbraz** [comment]
@a17r I'm fine with the jaraco-text change.

**Comment by gyakovlev** [comment]
ack weechat change. I can cherry pick it out if you want.

**Comment by a17r** [comment]
> ack weechat change. I can cherry pick it out if you want.

You can take it if you want it. Preferably I'd have someone from @gentoo/python ack the whole change.

**Comment by mgorny** [comment]
I think this is something you should do with per-package commits, so they're easier to revert.

**Comment by mgorny** [comment]
(and the number of packages is not high)

File: dev-python/billiard/billiard-3.6.0.0.ebuild [hunk 1]
```diff
@@ -30,10 +30,6 @@ DEPEND="
 DISTUTILS_IN_SOURCE_BUILD=1
 
 python_compile() {
```
Comment (mgorny) [inline dev-python/billiard/billiard-3.6.0.0.ebuild:32]: You can remove the whole phase.

File: dev-python/bsddb3/bsddb3-6.2.7.ebuild [hunk 2]
```diff
@@ -64,16 +64,9 @@ python_configure_all() {
 }
 
 python_compile() {
```
Comment (mgorny) [inline dev-python/bsddb3/bsddb3-6.2.7.ebuild:66]: Ditto.

File: dev-python/cytoolz/cytoolz-0.10.0.ebuild [hunk 3]
```diff
@@ -26,8 +26,6 @@ DEPEND="${RDEPEND}
 		>=dev-python/toolz-0.8[${PYTHON_USEDEP}] )"
 
 python_compile() {
```
Comment (mgorny) [inline dev-python/cytoolz/cytoolz-0.10.0.ebuild:28]: Likewise.

File: dev-python/fs/fs-2.4.9-r1.ebuild [hunk 4]
```diff
@@ -39,11 +39,7 @@ BDEPEND="test? (
 )"
 
 python_test() {
-	# python2_7 tests require dev-python/typing which is
-	# now in stdlib so ignore tests. py2.7 is going away.
-	if python_is_python3; then
-		esetup.py test || die "tests failed"
-	fi
+	esetup.py test || die "tests failed"
```
Comment (mgorny) [inline dev-python/fs/fs-2.4.9-r1.ebuild:42]: While touching it, maybe remove the unnecessary `||die`.

File: dev-python/numexpr/numexpr-2.6.9.ebuild [hunk 5]
```diff
@@ -41,11 +41,6 @@ python_prepare_all() {
 }
 
 python_compile() {
```
Comment (mgorny) [inline dev-python/numexpr/numexpr-2.6.9.ebuild:43]: Ditto.

File: dev-python/pytables/pytables-3.4.4.ebuild [hunk 6]
```diff
@@ -52,10 +52,6 @@ python_prepare_all() {
 }
 
 python_compile() {
```
Comment (mgorny) [inline dev-python/pytables/pytables-3.4.4.ebuild:54]: Likewise.

File: dev-python/sphinx-jinja/sphinx-jinja-0.3.0.ebuild [hunk 7]
```diff
@@ -26,9 +26,8 @@ DEPEND="${RDEPEND}
 		dev-python/sphinx-testing[${PYTHON_USEDEP}] )"
 
 python_prepare() {
```
Comment (mgorny) [inline dev-python/sphinx-jinja/sphinx-jinja-0.3.0.ebuild:28]: This ebuild sucks :-(.

**Comment by a17r** [comment]
> (and the number of packages is not high)

There'd probably be a lot more with multiline `python_gen_cond_dep` detection.

**Comment by ZeroChaos-** [comment]
ACK from Zero_Chaos

**Pull request merged**